tests/test_legacy_external_class.py

```diff
@@ -0,0 +1,63 @@
+from nose.tools import assert_true, assert_list_equal, raises
+from numpy.testing import assert_almost_equal
+import datajoint as dj
+from . import schema_legacy_external as modu
+
+
+def test_heading():
+    heading = modu.Simple().heading
```

Comment:
You removed the `()` in line 8 of `test_external_class.py` above. Should you do so here as well?

Reply:
This does not really matter.  Accessing attributes of classes directly still triggers instantiation of an object, so a more efficient way is to instantiate.  The omission of parentheses was provided for user convenience.